tests/integration/test_vulnerability_detector/test_scan_results/test_suse_inventory_suse_feed.py

```diff
@@ -74,6 +80,19 @@ def test_suse_vulnerabilities_report(get_configuration, configure_environment, r
 
     # Check the vulnerabilities of packages inserted
     for item in mock_vulnerability_scan['vulnerabilities']:
-        vd.check_vulnerability_scan_event(wazuh_log_monitor=wazuh_log_monitor, package=item['package']['name'],
-                                          cve=item['cve']['cveid'])
+        for packg in item['package']:
+            vulnerable = False
+            for deps in item['dependency']:
+                if deps['name'] in packg['name']:
```

Comment:
The `installed` flag already checks for this condition, so it is not necessary and can also be misleading.